src/Centreon/Application/DataRepresenter/ContactGroupEntity.php

```diff
@@ -74,7 +74,7 @@ public function jsonSerialize()
         return [
             'id' => $this->entity->getCgId(),
             'name' => $this->entity->getCgName(),
-            'activate' => $this->entity->getCgActivate()
+            'activate' => (int) $this->entity->getCgActivate()
```

Comment:
```suggestion
            'activate' => (int)$this->entity->getCgActivate()
```

For future compatibility ... https://www.php-fig.org/psr/psr-12/#61-unary-operators